backend/app/services/adapters/task_kinds/operations.py

```
@@ -1101,3 +1122,117 @@ def _log_cleanup_exception(task_or_future):
+        """
+        Schedule sending close-session event to a device.
+
+        This is a fire-and-forget operation that runs in background
+        and doesn't block task deletion.
+
+        Args:
+            user_id: User ID who owns the task
+            device_id: Device ID to send the event to
+            task_id: Task ID being deleted
+        """
+        import asyncio
```

Ruff (0.14.14)

[error] Redefinition of unused `asyncio` from line 11: `asyncio` redefined here

Remove definition: `asyncio`

(F811)